canonical-web-and-design/maas-ui
# feat(all): update dashboard URL to the React version

## Done

- Change the dashboard URLs to point to the React dashboard.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Log in as an admin.
- Load a react page and click on the header logo, you should go to the dashboard.
- Load a legacy page and click on the header logo, you should go to the dashboard.
- Log in as a non admin.
- Click on the header logo, you should go to the machine list.

## Fixes

Fixes: canonical-web-and-design/app-squad#140.
